hack/cpaas/patch-cluster-with-rhpm-0.1.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+declare -r PROJECT_ROOT
+declare -r powermon_bundle_package_name="power-monitoring-operator-bundle-container"
+declare -r OCP_VERSION=${OCP_VERSION:-'v4.13'}
+
+source "$PROJECT_ROOT/hack/utils.bash"
+
+function check_prereq {
+  header "checking pre requisites..."
+  if ! command -v jq &> /dev/null
+  then
+      err "jq could not be found. Please install jq first."
+      exit
+  fi
+  if ! command -v oc &> /dev/null
+  then
+      err "oc could not be found. Please install oc first."
+      exit
+  fi
+  if ! command -v podman &> /dev/null
+  then
+      err "podman could not be found. Please install podman first."
+      exit
+  fi
+  ok "all prereqs ok.."
+}
+
+function get_index_image {
+  local index_image="noimage"
+  local package=$1
+  local requested_ocp_version=$2
+  messages=$(curl -s 'https://datagrepper.engineering.redhat.com/raw?topic=/topic/VirtualTopic.eng.ci.redhat-container-image.index.built&delta=824000&contains='$package)
+
+  local i=0
+  local bundle=""
+  while true
```

Comment:
why not just capture the latest message? `curl -s 'https://datagrepper.engineering.redhat.com/raw?topic=/topic/VirtualTopic.eng.ci.redhat-container-image.index.built&delta=824000&contains=power-monitoring-operator-bundle-container' | jq '.raw_messages[0]'`

Reply:
to match the OCP version.